assets/js/main.js

```diff
@@ -227,12 +239,14 @@ function displayVeganData(data) {
 document.addEventListener('DOMContentLoaded', async function () {
     const id = getLocalStorage("userRecipes");
 
-    for (let i = 0; i < id.length; i++) {
-        try {
-            const data = await fetchData(id[i]);
-            displayRecipe(data);
-        } catch (error) {
-            console.error(error);
+    if (id !== null) {
```

Comment:
@quachies I added a condition here to see if `id` exists. It kept giving an error in the console, so wanted to make sure it does not happen. Let me know if it works!